### DISCUSSION TITLE
Clean uninstall / settings reference

### DISCUSSION THREAD

**Discussion by grantsut** [AutoDarkMode/Windows-Auto-Night-Mode#819] [Q&A] [answered]
Is there a list somewhere of all the places that Auto Dark Mode installs or changes files or settings? I have used it for several years and the installation was messed up, in that only some parts of the theme were being changed (I know there was an issue with Microsoft changing where theme settings were located). I have now uninstalled Auto Dark Mode, but the theme still changes automatically twice a day, but only partially. I want to remove all automatic changes and start again.

**Comment by Armin2208** [comment]
Possible it still changes, because you have the old tasks left in Task Scheduler. Open Task Scheduler and look for a folder named ADM_username.
![grafik](https://github.com/AutoDarkMode/Windows-Auto-Night-Mode/assets/32982430/2998b014-6888-4775-8f0d-bcc0a66eca54)

**Comment by grantsut** [comment]
There is no folder similar to that, or scheduled tasks that look like they could be relevant.

I couldn't find anything in the event viewer from around the time that the explorer windows changed color, but would anything be expected to show there?

**Reply by Armin2208** [reply, marked as answer]
We don't write anything into the Event Viewer.

Either you have an AutoDarkModeSvc.exe running in the background which you would see in Task Manager.
Or you have a Auto Dark Mode / Auto Night-Mode task on the Task Scheduler. On old versions we also put it right into the top folder of Task Scheduler.

If both don't apply, then you probably have a different app switching your theme. Maybe f.lux Or WinDynamicDesktop.

**Reply by grantsut** [reply]
It was f.lux, I didn't know that it could also (try to) change the Windows theme in addition to the colour temperature. The option is "use dark mode at sunset" if anyone else has this problem.

Thanks for your help!
